src/transforms/tree.js

```diff
@@ -158,10 +158,14 @@ function nodeData(field) {
 
 function normalizer(delimiter = "/") {
   return `${delimiter}` === "/"
-    ? P => P // paths are already slash-separated
+    ? P => P.map(pathJoin) // paths are already slash-separated
     : P => P.map(replaceAll(delimiter, "/")); // TODO string.replaceAll when supported
 }
 
+function pathJoin(p) {
+  return Array.isArray(p) ? p.map(str => `${str}`.replace(/\//g, '\\/')).join("/") : p;
+}
```

Comment:
I don’t think we should accept arrays. I think limiting the path to strings is simpler.